### PR TITLE
Fix document for testing on different platforms

### PR DESCRIPTION
**Pull request by khanhntd**
Update some notes and fix some variables and naming convention.

### CODE REVIEW

File: docs/run-testing-framework.md [hunk 1]
```diff
@@ -46,12 +46,12 @@ Setup only needs to be run once, it creates:
  
 Run 
 ````
-cd terraform/setup && terraform init && terraform apply
+cd terraform/setup && terraform init && terraform apply -auto-approve
```
Comment (sethAmazon) [inline docs/run-testing-framework.md:49]: Should we add the auto approve. Some people might want to look at the terraform plan.

Comment (sethAmazon) [reply docs/run-testing-framework.md:49]: IDC either way

**Review by sethAmazon — APPROVED** [review]

File: docs/run-testing-framework.md (hunk 1 above)
Comment (khanhntd) [reply docs/run-testing-framework.md:49]: IMO, mostly the behavior from Dev when joining the first time is making sure its work beforehand. Also the frequency using terraform plan is less than terraform apply/destroy with -auto-approve. Therefore, it should be good for us to add terraform apply/destroy -auto-approve with any worries.

File: docs/run-testing-framework.md [hunk 2]
```diff
@@ -60,11 +60,25 @@ Remember, if you have changes on sample apps or the mocked server, you need to r
  
 #### 2.1.3 Build Image
 Please follow https://github.com/aws-observability/aws-otel-collector/blob/main/docs/developers/build-docker.md to build your image with the new component, and push this image to dockerhub, record the image link, which will be used in your testing.
- 
-### 2.2 Run in ECS
+
+#### 2.2 Run in EC2
```
Comment (jefchien) [inline docs/run-testing-framework.md]: I'm a bit confused. Why did you move this up? It used to be 2.4. If you're going to move things around, then you need to fix the links to the sections in the README.

Comment (khanhntd) [reply docs/run-testing-framework.md]: Mostly for consistency with what being show [in the README](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/README.md#2-run-testing-framework). Also fixed the links. Nice catch

**Review by aateeqi — APPROVED** [review]

**Review by jefchien — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+CMD_DIR := $(abspath ./cmd)
+
+build-aotutil:
+	cd ${CMD_DIR}/aotutil && make build
```
Comment (jefchien) [inline Makefile:4]: Have you tested this?

Comment (khanhntd) [reply Makefile:4]: Yes. Tested it at local and also it would be easier to keep it consistent with [Soaking Github workflow ](https://github.com/aws-observability/aws-otel-collector/blob/main/.github/workflows/soaking.yml#L52-L53)
